README.txt

```diff
@@ -1,3 +1,7 @@
 gs is the command to run
 jq is a helper utility to parse JSON from GitHub
-See the comments at the top of the gs script file for more info
+Download jq for your system at https://stedolan.github.io/jq/
+Rename the executable 'jq' and place in the same directory as the gs script
```

Comment:
Oh yeah, I forgot that I actually put it in one of my path directories. Would it be better to just change places in the script where it calls jq to be ./jq? Then the user wouldn't have to modify the PATH variable.

Reply:
Actually PATH is preferable. Remember the PATH modification is only necessary if you want to use a custom jq binary in the same directory as the gs script. If you have properly installed jq on your computer then it probably is already accessible via PATH.

Also I think it’s standard to expect a unix tool dependency to exist on the PATH. If a unix tool depends on another unix tool it would be appropriate to expect the user to manage the dependency installation. For example, since I have jq on my Mac via brew install jq, I want to use that version when running this script. Depending on PATH to have a directory that contains jq gives freedom for me to supply whichever jq install I want. It would be similar if you wrote a script that uses wget. Some Linux distros don’t have wget installed by default; regardless your script would just expect it to be on the PATH.